### ISSUE TITLE
Missing "reason" in lib/rapi/client.py 

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#832]
Originally reported of Google Code with ID 776.

```
Not all the RAPI calls in lib/rapi/client.py expose the "reason" field.
It should be there.
```
Originally added on 2014-03-25 16:15:30 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-03-26 10:05:46 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in 2.8

commit 8ae4d0de7f08fdf374ba329e768b7b1bf46fe583
Author: Michele Tartara <mtartara@google.com>
Date:   Wed Mar 26 14:20:28 2014 +0100

    Add reason parameter to RAPI client functions
    
    Only the functions for starting, stopping and rebooting a VM had a reason
    parameter. Now, all the RAPI client functions generating opcodes do.
    
    Also, one test is expanded to verify that a RAPI request with both body and
    query supports the reason parameter properly.
    
    Fixes Issue 776.
    
    Signed-off-by: Michele Tartara <mtartara@google.com>
    Reviewed-by: Hrvoje Ribicic <riba@google.com>

```
Originally added on 2014-03-26 15:18:25 +0000 UTC.

Mentioned Issue with GoogleCode ID 776 has been migrated to GitHub with ID #832.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in 2.10.3.
```
Originally added on 2014-04-16 10:37:20 +0000 UTC.

Changed State: Released
